modules/apps/simplicity-theme/src/js/main.js

```diff
@@ -0,0 +1,24 @@
+window.onscroll = function() {checkOffset()};
+
+var navigation = document.getElementById("navigation-top");
+var navigationOffsetTop = navigation.offsetTop;
+
+function checkOffset() {
+  if (window.pageYOffset >= navigationOffsetTop) {
+    navigation.classList.add("navigation-offset-top");
+  }
+  else {
+    navigation.classList.remove("navigation-offset-top");
+  }
+}
+
+function checkNavBarToggled() {
+	var navBar = document.getElementById("navigation-top");
```

Comment:
Since we are lookup the same element (`navigation-top`) we should cache the first lookup and reuse the JS variable rather than looking it up again.